### PR TITLE
fix: target actor unique ID serialization

### PR DESCRIPTION
**Pull request by nicholass003**
Fixes https://github.com/pmmp/BedrockProtocol/issues/268

### CODE REVIEW

**Comment by nicholass003** [comment]
Tested and working fine

https://github.com/user-attachments/assets/2c406434-63cf-4134-ab62-2924321fec4e

File: src/types/camera/CameraTargetInstruction.php [hunk 1]
```diff
@@ -21,24 +21,24 @@ final class CameraTargetInstruction{
 
 	public function __construct(
 		private ?Vector3 $targetCenterOffset,
-		private int $actorUniqueId
+		private int $targetActorUniqueId //This is a little-endian long, NOT a var-long. (WTF Mojang)
```
Comment (IvanCraft623) [inline src/types/camera/CameraTargetInstruction.php:24]: No need to change the property name

Comment (IvanCraft623) [inline src/types/camera/CameraTargetInstruction.php:24]: This command belongs to de/serealization, not the constructor

**Review by IvanCraft623 — CHANGES_REQUESTED** [review]
@TwistedAsylumMC or @dries-c Can you confirm this?

**Review by TwistedAsylumMC — DISMISSED** [review]

**Review by dries-c — APPROVED** [review]

**Review by IvanCraft623 — APPROVED** [review]

**Review by dries-c — APPROVED** [review]

**Pull request merged**